src/client/components/Registration.jsx

```diff
@@ -1,4 +1,44 @@
+import { useState } from "react";
+
+
+const emptyUser = {
+    email: "",
+    username: "",
+    password: ""
+}
+
+const registerURL = "http://localhost:4000";
```

Comment:
need to ask steve/nathan about importing urls from .env

Reply:
This does need improvement. I left it like that for reference. "importing" it from the env file will be the way to go i feel. we just need to discuss it with Steve.

Reply:
Yup you can setup production and development URLs in there ;) Might do a workshop on this...